src/Stratis.Bitcoin.IntegrationTests.Common/Runners/BitcoinCoreRunner.cs

```diff
@@ -18,21 +21,48 @@ public BitcoinCoreRunner(string dataDir, string bitcoinDPath)
 
         public new bool IsDisposed
         {
-            get { return this.process == null && this.process.HasExited; }
+            get { return this.process?.HasExited ?? true; }
```

Comment:
If `HasExited` is false then `IsDisposed` can return `false` and the same for `true`.  

Does this need the `??` operator ?